openshift/templates/identity-kit-admin/identity-kit-admin-deploy.ns.param

```diff
@@ -0,0 +1,28 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/identity-kit-admin/identity-kit-admin-deploy.json
+#=========================================================
+NAME=ns-id-admin
+APP_NAME=ns-id
+APP_GROUP=ns-id
+SUFFIX=
```

Comment:
Should a suffix be defined to ensure resources are created with unique names?

Reply:
I think the name of the Nova Scotia instance is different enough from the base app that it is not necessary (I wouldn't really be sure what suffix to assign without making the naming weird to be honest).